### PR TITLE
update to sorting function (def sort_treeview)

### PR DESCRIPTION
**Pull request by Botnen1**
# Description

With this change, the conversion to integers happens once per item, rather than potentially multiple times per item during the sort comparisons. For string-wise sorting, it defaults to the natural sort behavior of lists in Python, which is string-wise.
Fixes # (issue)

### CODE REVIEW

File: Main.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from tkinter import ttk, filedialog
 from os.path import exists
 from os import listdir
+import customtkinter as ctk
```
Comment (Kubis10) [inline Main.py:9]: Why is this import needed?

**Review by Kubis10 — APPROVED** [review]
Thanks for commiting, it will help to clear out this function

File: Main.py (hunk 1 above)
Comment (Botnen1) [reply Main.py:9]: oh no im so sorry that import is not needed
i had an idea for some other functionallity but i forgot to remove the import

Comment (Botnen1) [reply Main.py]: i commited again after removing the import
